dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+#include "pch.h"
+
+const std::wstring ConvertByteArrayToWideString(unsigned int payloadLength, _In_reads_(payloadLength) byte* payload);
+void ProtocolLaunchHelper(unsigned int payloadLength, _In_reads_(payloadLength) byte* payload);
+
+inline HRESULT GetAppUserModelId(wil::unique_cotaskmem_string& appUserModelId)
```

Comment:
Commnet from prior PR - should this method just return a cotaskmemstring instead of emitting it as a result value?

```c++
wil::unique_cotaskmem_string GetAppUserModelId()
{
    wchar_t appId[APPLICATION_USER_MODEL_ID_MAX_LENGTH + 1] = {};
    UINT32 appIdSize { ARRAYSIZE(appId) };
    const HRESULT hr = ::GetCurrentApplicationModelUserId(&appIdSize, appId);
    if (hr != HRESULT_FROM_WIN32(ERROR_NO_PACKAGE_IDENTITY)) // Not sure this is the right error
        THROW_IF_FAILED(hr);
    return wil::make_unique_string<wil::unique_cotaskmem_string>(appId);
}
```

Then at the callsite:

```c++
if (auto appId = GetAppUserModelId()) 
{
    // Process has identity
}
```